packages/web/src/modules/core/styles/switchStyle.ts

```diff
@@ -0,0 +1,55 @@
+import { styled } from "../../theming/themes"
+const size = 30
+const color = "#009fb7"
+
+export const LabelSwitch = styled.label`
+    position: relative;
+    display: inline-block;
+    width: ${size * 2}px;
+    height: ${size}px;
+`
+export const InputSwitch = styled.input`
+    & {
+        opacity: 0;
+        width: 0;
+        height: 0;
+    }
```

Comment:
The `& { }` block is useless

Reply:
Almost all of them are gone, some are still required though